### ISSUE TITLE
[Question] How to update "nightly" rvgl?

### ISSUE THREAD

**Issue by 7wells** [Calinou/scoop-games#181]
Hello,

rvgl has been updated on 29 Dec 2020. I have an older version installed via the official "rvgl.json" file, which says the updates are "nightly". When I do `scoop update rvgl`, it tells me:
> rvgl: nightly-20200821 (latest version)

However, this is not the latest version, which is version 20.1230, dated 29 Dec 2020:
https://distribute.re-volt.io/releases/

The json file points to this link:
https://distribute.re-volt.io/releases/rvgl_full_win64_online.zip

When I manually download and look into it, it is indeed the latest version 20.1230, but that does not help, as scoop thinks that `nightly-20200821` is the latest version.

How shall users update such "nightly" programmes/games? Shall I uninstall rvgl first and then install it (again)? Will settings be retained or lost?

Thanks for your help and my best wishes to all of you for 2021!

**Comment by Calinou** [comment]
> How shall users update such "nightly" programmes/games? Shall I uninstall rvgl first and then install it (again)? Will settings be retained or lost?

Yes, that's currently the only way to update `nightly` apps in Scoop. Your data will be persisted in the `persist` folder so you shouldn't lose it.

**Comment by 7wells** [comment]
Thanks for your quick confirmation. Rvgl has no folder under "persist", though. I will make a backup of "everything" rvgl-related to play it safe.

**Comment by 7wells** [comment]
Hello Calinou, I do not understand why it does not work. Currently, the json is this:

```
{
    "homepage": "https://rvgl.re-volt.io/",
    "description": "Modern reimplementation of the classic RC racing game Re-Volt (full package with extras)",
    "version": "nightly",
    "license": "Freeware",
    "architecture": {
        "64bit": {
            "url": "https://distribute.re-volt.io/releases/rvgl_full_win64_online.zip"
        },
        "32bit": {
            "url": "https://distribute.re-volt.io/releases/rvgl_full_win32_online.zip"
        }
    },
    "shortcuts": [
        [
            "rvgl.exe",
            "RVGL"
        ]
    ]
}
```

When I go to the link (64bit), it is the correct file (latest, i.e. 20.1230 version, dated 29 Dec 2029). But even after completely uninstalling rvgl via `scoop uninstall rvgl` (and making sure that there are no "left-over" folders and files) and doing a fresh `scoop install rvgl` and then `scoop update rvgl`, it still tells me: `rvgl: nightly-20200821 (latest version)`
How come that scoop thinks that nightly-20200821 is the latest version? What am I missing?

Thanks for your insights - and my best wishes to you for 2021!

**Comment by Calinou** [comment]
@7wells I don't know why that would happen, sorry.

**Comment by beyondmeat** [comment]
@Calinou  can close, scoop requires --force for nightlies.

**Comment by beyondmeat** [comment]
@7wells please close
